Sources/ArcGISToolkit/Components/Bookmarks/BookmarksHeader.swift

```diff
@@ -47,15 +47,15 @@ struct BookmarksHeader: View {
                 maxWidth: .infinity,
                 alignment: .leading
             )
-            if horizontalSizeClass == .compact || verticalSizeClass == .compact {
-                Spacer()
-                Button {
-                    isPresented.toggle()
-                } label: {
-                    Text.done
-                        .fontWeight(.semibold)
-                }
+            Spacer()
+            Button {
```

Comment:
It is nice to get system styling where we can but I'm not sure I see where I'd fit a `NavigationStack` into the component's hierarchy, especially since it doesn't present another view. Any suggestion?

Reply:
Well, it would have to be at the root of the `Bookmarks` view and the navigation bar would essentially replace the `BookmarksHeader` view. But the styling would be difference; not sure whether that would be a deal breaker.

Reply:
I've spent some more time experimenting with incorporating a `NavigationStack`. My feeling is that it's going to take some wrangling to get it integrated nicely. My preference is to leave the header as is but we can wait for Mark to weigh in.

Reply:
I think If we wanted to chase that further, it should be done separately and not hold up this PR.

Reply:
Agreed. I didn't mean to suggest that it needed to be addressed in this PR. I was just pointing out a potential improvement.